I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 61, in script
    creature "--They drugged you. And now you are my sacrifice. And I probabbly will kill you. Perhaps not. There are worse destines than death. But the way I will inflict pain to you will depends of your answers."
IOError: Couldn't find file 'malignchords.ogg'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 61, in script
    creature "--They drugged you. And now you are my sacrifice. And I probabbly will kill you. Perhaps not. There are worse destines than death. But the way I will inflict pain to you will depends of your answers."
  File "renpy/ast.py", line 716, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "renpy/exports.py", line 1417, in say
    who(what, *args, **kwargs)
  File "renpy/character.py", line 1219, in __call__
    self.do_display(who, what, cb_args=self.cb_args, dtt=dtt, **display_args)
  File "renpy/character.py", line 871, in do_display
    **display_args)
  File "renpy/character.py", line 621, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "renpy/ui.py", line 298, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "renpy/display/core.py", line 3271, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, **kwargs)
  File "renpy/display/core.py", line 3971, in interact_core
    renpy.audio.audio.periodic()
  File "renpy/audio/audio.py", line 1138, in periodic
    raise_(exc[0], exc[1], exc[2])
  File "renpy/audio/audio.py", line 1116, in periodic_thread_main
    periodic_pass()
  File "renpy/audio/audio.py", line 1058, in periodic_pass
    c.periodic()
  File "renpy/audio/audio.py", line 512, in periodic
    topf = load(filename)
  File "renpy/audio/audio.py", line 69, in load
    rv = renpy.loader.load(fn)
  File "renpy/loader.py", line 798, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'malignchords.ogg'.

Windows-10-10.0.19041
Ren'Py 7.4.7.1762
sacrifice 1.0
Mon Jul 19 18:30:56 2021
